examples/sls-apps-monorepo/ize.toml

```diff
@@ -0,0 +1,22 @@
+terraform_version = "1.2.6"             # (optional) Terraform version can be set here. 1.1.3 by default
+# prefer_runtime = ""                 	# (optional) Prefer a specific runtime. (native or docker) (default 'native')
+# tag = ""                            	# (optional) Tag can be set statically. Normally it is being constructed automatically based on the git revision.
+# plain_text = false                  	# (optional) Plain text output can be enabled here. Default is false. Can be overridden by IZE_PLAIN_TEXT env var or --plain-text flag.
+# env = "dev"                         	# (optional) Environment name can be specified here. Normally it should be passed via `ENV` variable or --env flag.
+# env_dir = ""                        	# (optional) Environment directory can be specified here. Normally it's calculated automatically based on the directory structure convention.
+# docker_registry = ""                	# (optional) Docker registry can be set here. By default it uses ECR repo with the name of the service.
+# tf_log_path = ""                    	# (optional) TF_LOG_PATH can be set here.
+# custom_prompt = false               	# (optional) Custom prompt can be enabled here for all console connections. Default: false.
+# aws_profile = ""                    	# (optional) AWS Profile can be specified here (but normally it's specified via AWS_PROFILE env var)
+# log_level = ""                      	# (optional) Log level can be specified here. Possible levels: info, debug, trace, panic, warn, error, fatal(default). Can be overridden via IZE_LOG_LEVEL env var or via --log-level flag.
+# ize_dir = ""                        	# (optional) Ize directory can be specified here. Normally it's assumed to be .infra or .ize in the current repo.
+# apps_path = ""                      	# (optional) Path to apps directory can be set. By default apps are searched in 'apps' and 'projects' directories. This is needed in case your repo structure is not purely ize-structured (let's say you have 'src' repo in your dotnet app, as an example)
+# root_dir = ""                       	# (optional) Project directory can be set here. By default it's the current directory, but in case you prefer to run ize from the outside of repo it may be useful (uncommon).
+# tf_log = ""                         	# (optional) Terraform TF_LOG can be set here.  Can be TRACE, DEBUG, INFO, WARN or ERROR.
+# config_file = ""                    	# (optional) Path to ize.toml config file can be specified, but normally it's read from the environment's directory automatically.
+# home = ""                           	# (optional) User home directory can be specified here. Normally $HOME is used.
+
+[terraform.infra]
+root_domain_name = ""   	# (optional) Root domain name can be set here. This is the main domain that will be passed to the terraform. Generally if your app lives at 'api.dev.nutcorp.net' the root domain is `nutcorp.net`
```

Comment:
Let's not use it as an example. If you are trying to use it b/c of toml parsing, let's not show this workaround in the ize.toml please (and better fix the issue later), as it's a bit confusing and raises "Why is it set to an empty string here?" questions.
Also, please fix the indent of the comment so it aligned